src/components/MealForm/MealForm.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components/macro'
+import Input from '../Input/Input'
+
+export default function MealForm() {
+  return (
+    <MealFormWrapper>
+      <Input
```

Comment:
I'd encourage you to change the HTML element to an `input` :)

Reply:
Already done.